src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -165,7 +165,7 @@ class HuiGenericEntityRow extends LitElement {
       .info {
         margin-left: 16px;
         margin-right: 8px;
-        flex: 1 0 30%;
+        flex: 1 0 60px;
```

Comment:
Why don't we allow it to shrink?

Reply:
@bramkragten Shrink can help too so forcepushed with "shrink".